src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/benchmarks.wcfcore.Perf.yml

```diff
@@ -1,83 +1,59 @@
 jobs:
-  BasicHttpWithBuffered:
+  WCFCorePerf:
     source:
       repository: https://github.com/dotnet/wcf.git
       branchOrCommit: master
       project: src/System.Private.ServiceModel/tests/Benchmarks/WCFCorePerf/WCFCorePerf/WCFCorePerf.csproj
     isConsoleApp: true
     waitForExit: true
```

Comment:
You might want to add `readyStateText` with some text that is displayed by the app when it's ready, such that the Agent knows when it can start the load, and also measure the startup time.

Reply:
I have updated the changes in commit [8656748](https://github.com/dotnet/wcf/pull/4394/commits/86567482341bdebca10c0d99d0306626ffa07913)